easybuild/easyconfigs/b/beagle-lib/beagle-lib-3.1.2-GCC-10.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'beagle-lib'
+version = '3.1.2'
+
+homepage = 'https://github.com/beagle-dev/beagle-lib'
+description = """beagle-lib is a high-performance library that can perform the core calculations at the heart of most
+ Bayesian and Maximum Likelihood phylogenetics packages."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/beagle-dev/beagle-lib/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd872b484a3a9f0bce369465e60ccf4e4c0cd7bd5ce41499415366019f236275']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+]
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('pkg-config', '0.29.2'),
+    ('pocl', '1.8'),
+]
+
+preconfigopts = "./autogen.sh && "
+configopts = "--with-opencl=$EBROOTPOCL --with-cuda=$EBROOTCUDA "
```

Comment:
@stevenvdb was a bit surprised by this addition, given there are no other references to CUDA in the dependencies?

Reply:
I copy-pasted this from `beagle-lib-3.1.2-GCC-10.2.0.eb`. According to the installation log, the `--with-cuda` option is ignored because `$EBROOTCUDA` is not set, so I removed it.